setup.py

```diff
@@ -6,9 +6,9 @@
 
 from setuptools import setup
 
-if not sys.version_info[:2] >= (3, 7):
+if not sys.version_info[:2] >= (3, 8):
     sys.exit(
-        f"conda is only meant for Python 3.7 and up. "
+        f"conda is only meant for Python 3.8 and up. "
```

Comment:
How about "conda requires a Python 3.8 or newer runtime" or "conda is supported in Python 3.8 and up"